tailwind.config.js

```diff
@@ -48,8 +48,8 @@ export default {
                 },
 
                 border: {
-                    emphasis: color('--border-emphasis', colors.slate[500]),
-                    DEFAULT: color('--border', colors.slate[200]),
+                    emphasis: color('--border-emphasis', colors.slate[400]),
+                    DEFAULT: color('--border', colors.slate[300]),
```

Comment:
Why? Do we want this everywhere?

Reply:
The previous color was way too light, border can have a bit more contrast

Reply:
Please within a separated PR and to all places where it's used; core, docs, blade-components, etc.